docs/guides/nrfconnect_examples_configuration.md

```diff
@@ -76,61 +79,55 @@ type. Example on how to use config options was presented below:
-    to override their configuration values in the main application config file,
-    so you will not have to modify their files unless you are interested in
-    developing those libraries.
-
--   **Application configuration file**
-
-    Every example application has its main configuration file called `prj.conf`
-    located in the example directory. It typically contains application specific
-    configuration or most frequently changed options, but actually almost every
-    configuration can be overrided in this file and probably this file is the
-    most important for the User.
-
--   **Overlays**
-
-    Overlay files are usually used to extract configuration for some specific
-    case or feature from the general application configuration. The main
-    difference between them and the application `prj.conf` file is that they are
-    not included automatically, so User may decide whether to build sample with
-    or without them.
-
--   **Board configuration files**
-
-    These are hardware platform dependent configuration files, which are
-    automatically included basing on the compilation target board name and they
-    contain board and its peripherals configuration.
-
-Visit
-[Kconfig](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/kconfig/index.html#kconfig)
-if you are interested in getting more advanced and detailed information about
-used configuration structure.
+The application configuration is specified using Kconfig files.
+Typically, there are many files having impact on the final configuration shape,
+because most of the components have their own configuration files.
+
+There is no need to modify all these files separately.
+See the following list for types of files you can find in the project
+and which of them are important from your perspective:
+
+- **Software libraries' configuration files.**
+  Many libraries, components and modules have their own configuration files,
+  but in the majority of cases you can override their configuration values
+  in the main application configuration file, meaning that you do not need to modify
+  their files unless you are interested in developing these libraries.
+
+- **Application configuration file.**
+  Every example application has its main configuration file called `prj.conf` that is
+  located in the example directory. This file contains application-specific configuration
+  or the most frequently changed options. Almost every configuration can be overridden
+  in this file and probably this file is the most important one.
+
+- **Overlays.**
+  Overlay files are usually used to extract configuration for some specific
+  case or feature from the general application configuration. The main
+  difference between them and the application `prj.conf` file is that they are
+  not included automatically, so you can decide whether to build sample with
+  or without them.
+
+- **Board configuration files.**
```

Comment:
How about describing board configuration at the beginning and then othr configuration files which override the previous configuration files?